src/main/java/com/cag/cagbackendapi/service/organization/OrganizationService.java

```diff
@@ -0,0 +1,18 @@
+package com.cag.cagbackendapi.service.organization;
+
+import org.slf4j.Logger;
+import org.springframework.stereotype.Service;
+
+@Service
+public class OrganizationService {
+
+    private final Logger logger;
```

Comment:
I have never seen a final method before is it preferred to be used over static and void types of methods?

Reply:
This is a field of the OrganizationService! Final just means it will stay a constant value - immutable. If I make a final string and set it to some value, I will get an error if I try and change it to a different value later in my code. The Logger we use can be final because it will not need to change once it's instantiated.